app/livechat/imports/server/rest/dashboards.ts

```diff
@@ -13,29 +11,25 @@ import {
 	getAgentsProductivityMetrics,
 	getChatsMetrics,
 } from '../../../server/lib/analytics/dashboards';
-import { Users } from '../../../../models';
+import { Users } from '../../../../models/server';
 
 API.v1.addRoute(
 	'livechat/analytics/dashboards/conversation-totalizers',
 	{ authRequired: true },
 	{
-		get() {
+		async get() {
 			if (!hasPermission(this.userId, 'view-livechat-manager')) {
 				return API.v1.unauthorized();
 			}
 			let { start, end } = this.requestParams();
 			const { departmentId } = this.requestParams();
 
-			check(start, String);
-			check(end, String);
-			check(departmentId, Match.Maybe(String));
-
-			if (isNaN(Date.parse(start))) {
+			if (!start) {
 				return API.v1.failure('The "start" query parameter must be a valid date.');
 			}
 			start = new Date(start);
 
-			if (isNaN(Date.parse(end))) {
+			if (!end) {
```

Comment:
When doing these changes, look at what the code was doing before. Here, this if will just throw an error if the param is not present, but not if the param is "fasdfasdf" instead of a date 👀

Reply:
Understood, also brought them back in this commit: 5b3bc8b74949a9350d52c3d9f288bd809bbcdf02